docs/application_packages.md

```diff
@@ -464,8 +464,8 @@ if you have the following in src/main/resources/log4j.properties:
 The root logger’s level is set to WARN and the output is set to the console (stdout).
 
 Note that by default from project created from the maven archetype,
-there is already a log4j.properties file under src/test/resources and
-that file is only used for the unit test.
+there is already a log4j.props file under src/main/resources that can be modified or replaced.
+Also there is log4j.properties file under src/test/resources and that file is only used for the unit test.
```

Comment:
"Also there is a log4j.properties file under src/test/resources and that file is only used for unit tests."